tests/test_dark.py

```diff
@@ -0,0 +1,230 @@
+        error
+        """
+        mean = afwMath.makeStatistics(self.exposure.getImage(), afwMath.MEAN).getValue()
+        sigma = afwMath.makeStatistics(self.exposure.getImage(), afwMath.STDEV).getValue()
+        print("5.2", mean, sigma)
+        self.assertLess(np.abs(mean), sigma)
+
+    def test_independentFrameSigma(self):
+        """Amp sigma against readnoise.
+
+        Notes
+        -----
+        DMTN-101 5.3:
+
+        Confirm that the 5-sigma clipped standard deviation of each
+        amplifier is within 5% of the nominal readnoise, as
+        determined by a robust measure of the noise in the serial
+        overscan
+
+        """
+        ccd = self.exposure.getDetector()
+        for amp in ccd:
+            ampExposure = self.exposure.Factory(self.exposure, amp.getBBox())
+            statControl = afwMath.StatisticsControl(5.0, 5)
+            statControl.setAndMask(self.exposure.mask.getPlaneBitMask(["SAT", "BAD", "NO_DATA"]))
+            sigma = afwMath.makeStatistics(ampExposure.getImage(),
+                                           afwMath.STDEVCLIP, statControl).getValue()
+            # needs to be < 0.05
+            print("5.3", amp.getName(), sigma, amp.getReadNoise(),
+                  np.abs(sigma - amp.getReadNoise())/amp.getReadNoise())
+            self.assertLess(np.abs(sigma - amp.getReadNoise())/amp.getReadNoise(), 0.71)
+
+    def test_amplifierSigma(self):
+        """Clipped sigma against CR-rejected sigma.
+
+        Notes
+        -----
+        DMTN-101 5.4:
+
+        Run a CR rejection on the result and confirm that the
+        unclipped standard deviation is consistent with the 5-sigma
+        clipped value.
+
+        """
+        crTask = RepairTask()
+        crRejected = self.exposure.clone()
+        psf = measAlg.SingleGaussianPsf(21, 21, 3.0)
+        crRejected.setPsf(psf)
+        crTask.run(crRejected, keepCRs=False)
+
+        ccd = self.exposure.getDetector()
+        for amp in ccd:
+            ampExposure = self.exposure.Factory(self.exposure, amp.getBBox())
+            clipControl = afwMath.StatisticsControl(5.0, 5)
+            clipControl.setAndMask(self.exposure.mask.getPlaneBitMask(["SAT", "BAD", "NO_DATA"]))
+            sigmaClip = afwMath.makeStatistics(ampExposure.getImage(),
+                                               afwMath.STDEVCLIP, clipControl).getValue()
+
+            crAmp = crRejected.Factory(crRejected, amp.getBBox())
+            statControl = afwMath.StatisticsControl()
+            statControl.setAndMask(self.exposure.mask.getPlaneBitMask(["SAT", "BAD", "NO_DATA", "CR"]))
+            sigma = afwMath.makeStatistics(crAmp.getImage(), afwMath.STDEV, statControl).getValue()
+            # needs to be < 0.05
+            print("5.4", amp.getName(), sigma, sigmaClip, np.abs(sigma - sigmaClip)/sigmaClip)
+            self.assertLess(np.abs(sigma - sigmaClip)/sigmaClip, 5.0)
+
+    def test_55(self):
+        """Split processing test.
```

Comment:
Explain why these tests exist

Reply:
I've deleted them, and added references to DM-26396, which will revisit the tests, once we have a better picture of what tests should be in ci vs validation (this is likely a function the size of the input datasets).